tests/test_odm2/test_readservice.py

```diff
@@ -183,8 +185,11 @@ def test_getRelatedModelsByCode(self):
         assert len(resapi) == 0
 
         # test invalid argument
-        resapi = self.reader.getRelatedModels(code = models.ActionBy)
-        assert resapi is None
+        resapi = self.reader.getRelatedModels(code = 234123)
```

Comment:
> You can use `assert not a` that will be valid for both `[]` and `None`.

What do you mean by this? What's an example? Sorry, I'm a noob when it comes to assertion multiple things.